### DISCUSSION TITLE
Service Template Request: Grist

### DISCUSSION THREAD

**Discussion by rorywohl** [coollabsio/coolify#4688] [Service Template Requests]
Grist is a modern relational spreadsheet that combines the flexibility of a spreadsheet with the robustness of a database.

The `grist-core` repo is open source and provides the foundation for Grist Labs's commercial product.

`grist-core` - https://github.com/gristlabs/grist-core
"Self-Managed Grist" - https://support.getgrist.com/self-managed/
Grist docs - https://support.getgrist.com/

**Comment by Jeto143** [comment]
So looks like this got implemented, but if anyone here attempts to set it up with Pocket ID as an OIDC provider (it would probably happen with different ones too), they'll probably encounter a bug ("option domain is invalid" in the logs when trying to sign in). To fix it, replace:

```
- GRIST_DOMAIN=${SERVICE_URL_GRIST}
```

with:

```
- GRIST_DOMAIN=${SERVICE_FQDN_GRIST}
```

in the service's environment variables, then restart.

(I might create a PR to fix [the template](https://github.com/coollabsio/coolify/blob/v4.x/templates/compose/grist.yaml) later, but for now I just wanna help anyone stumbling upon the same issue.)

**Comment by bdmorin** [comment]
The template requires you to use OIDC, and not SAML. 

The template is very confusing.

**Comment by ShadowArcanist** [comment]
Added on https://github.com/coollabsio/coolify/pull/5212
